modules/phase_field/src/materials/SwitchingFunctionMaterial.C

```diff
@@ -46,6 +52,14 @@ SwitchingFunctionMaterial::computeQpProperties()
       _prop_d2f[_qp] = n * (120.0 * n * n - 180.0 * n + 60.0);
       break;
 
+    case 2: // MIX
+      _mix_weight = _mix_weight > 1 ? 1 : (_mix_weight < 0 ? 0 : _mix_weight);
```

Comment:
No. Range limit the parameter, set it in the initializer list, and naver change it again. In fact make it `const` in the declaration.